docs/i18n/japanese/how-to-help-with-video-challenges.md

```diff
@@ -80,61 +80,60 @@ videoId: nVAaxZ34khk
 ---
 ```
 
-Next, access the YouTube video with that `videoId`. The URL for the video will be:
-https://www.youtube.com/watch?v=[videoId] (replace `videoId` in the URL with the video's ID - without square brackets)
+次に、その「videoId」で YouTube の動画にアクセスします。 ビデオの URL は次のとおりです。
+https://www.youtube.com/watch?v=[videoId] (URL 中の「videoId」をビデオの ID に置き換えます。角括弧は不要。)
 
-In the example above, the URL is https://www.youtube.com/watch?v=nVAaxZ34khk
+上記例では、URL は https://www.youtube.com/watch?v=nVAaxZ34khk です。
 
-Skim the YouTube video with that videoId and think of a multiple-choice question based on the content of the video.
+その videoId でYouTube ビデオに目を通し、ビデオコンテンツに基づいて多肢選択式の質問を考えてください。
 
-### Add the question to the markdown file
+### マークダウンファイルに質問を追加してください
+ローカルでまたは GitHub インターフェースを使用して質問を追加することができます。 ローカルで質問を追加するには、「freeCodeCamp をローカルに設定する」(how-to-setup-freecodecamp-locally.md) 必要があります。 GitHub でファイルを見つけて、編集ボタンをクリックして、ブラウザで質問を追加することもできます。
```

Comment:
This line does not match with the line number in the source file. I checked Crowdin and the translation was missing a blank line in it. So I added one.
https://freecodecamp.crowdin.com/translate/457d487b1586f10d2fba3c4ccb9c5453/5360/en-ja/106#721946

Reply:
This is fixed now.